CHANGELOG/CHANGELOG-1.13.md

```diff
@@ -0,0 +1,89 @@
+- Add a new Prometheus metric to represent the number of packets dropped by OVS meter. ([#5165](https://github.com/antrea-io/antrea/pull/5165), [@mengdie-song])
+- Add support for the `sort-by` flag in more `antctl get` commands for more fields. ([#4346](https://github.com/antrea-io/antrea/pull/4346), [@jainpulkit22])
+- Add `kubeAPIServerOverride` option to allow user to override the address of Kubernetes apiserver for antrea-controller. ([#5056](https://github.com/antrea-io/antrea/pull/5056), [@tnqn])
+- Add a configuration toggle for FlowExporter to prevent Antrea Agent from showing error log `Error when initializing flow exporter` when user enable FlowExporter feature gate without installing Flow Aggregator. ([#5021](https://github.com/antrea-io/antrea/pull/5021), [@yuntanghsu])
+- Add documentation for deploying Antrea with a Rancher cluster. ([#4733](https://github.com/antrea-io/antrea/pull/4733), [@jainpulkit22])
+- Add OpenAPI schema for AntreaAgentInfo and AntreaControllerInfo CRDs. ([#5206](https://github.com/antrea-io/antrea/pull/5206), [@ceclinux])
+
+### Changed
+
+- Multiple APIs are promoted from alpha to beta.
+  - Promote ClusterGroup and Group to v1beta1. ([#5181](https://github.com/antrea-io/antrea/pull/5181), [@GraysonWu])
+  - Promote ExternalIPPool API to v1beta1. ([#5176](https://github.com/antrea-io/antrea/pull/5176), [@hongliangl])
+  - Promote Tier API to v1beta1. ([#5172](https://github.com/antrea-io/antrea/pull/5172), [@GraysonWu])
+  - Promote Egress API to v1beta1. ([#5180](https://github.com/antrea-io/antrea/pull/5180), [@wenqiq])
+  - Promote Upgrade AntreaClusterNetworkPolicy and AntreaNativeNetworkPolicy API to v1beta1. ([#5186](https://github.com/antrea-io/antrea/pull/5186), [@GraysonWu])
+  - Promote Traceflow API to v1beta1. ([#5108](https://github.com/antrea-io/antrea/pull/5108), [@luolanzone])
+- Add command `antctl upgrade api-storage` to support resource storage version migration. ([#5198](https://github.com/antrea-io/antrea/pull/5198), [@hongliangl])
+- Support DSR mode for Service's external addresses including LoadBalancerIPs and ExternalIPs on AntreaProxy, and document LoadBalancerMode related feature gate and configuration. ([#5202](https://github.com/antrea-io/antrea/pull/5202) [#5251](https://github.com/antrea-io/antrea/pull/5251), [@tnqn])
+- Enhance L7 NetworkPolicy to support TLS protocol. ([#4932](https://github.com/antrea-io/antrea/pull/4932), [@hongliangl])
+- Enable `proxyAll` by default for AntreaProxy on Windows because the kube-proxy userspace datapath has been removed since kubernetes 1.26. ([#4980](https://github.com/antrea-io/antrea/pull/4980), [@XinShuYang])
+- Support `service.kubernetes.io/service-proxy-name` label in AntreaProxy to align with KEP 2447. ([#4973](https://github.com/antrea-io/antrea/pull/4973), [@hongliangl])
```

Comment:
the `service.kubernetes.io/service-proxy-name` label in AntreaProxy

This one should be "Added" too?